api/WheelEvent.json

```diff
@@ -50,45 +50,47 @@
           "deprecated": false
         }
       },
-      "deltaMode": {
+      "on_window": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/WheelEvent/deltaMode",
+          "description": "<code>window</code> returns <code>WheelEvent</code>",
           "support": {
             "webview_android": {
-              "version_added": null
+              "version_added": true
             },
             "chrome": {
-              "version_added": null
+              "version_added": true
             },
             "chrome_android": {
-              "version_added": null
+              "version_added": true
             },
             "edge": {
-              "version_added": null
+              "version_added": true
             },
             "edge_mobile": {
-              "version_added": null
+              "version_added": true
             },
             "firefox": {
-              "version_added": null
+              "version_added": "17"
             },
             "firefox_android": {
-              "version_added": null
+              "version_added": "17"
             },
             "ie": {
-              "version_added": null
+              "version_added": "9"
             },
             "opera": {
-              "version_added": null
+              "version_added": true
             },
             "opera_android": {
-              "version_added": null
+              "version_added": true
             },
             "safari": {
-              "version_added": null
+              "version_added": true,
+              "notes": "Safari does not support <code>WheelEvent</code> but returns an object from <code>window.WheelEvent</code> nonetheless."
             },
             "safari_ios": {
-              "version_added": null
+              "version_added": true,
```

Comment:
Although it's probably safe to assume `true`, I'm seeing `safari_ios` as `null`.

Reply:
Seeing as it returns an object from `window.WheelEvent` but does not actually work. I think marking it `false` and adding a note is a better way to go.